server/plugin/api.go

```diff
@@ -56,6 +56,18 @@ type PRDetails struct {
 	Reviews            []*github.PullRequestReview `json:"reviews"`
 }
 
+type FilteredNotification struct {
+	github.Notification
+	HTMLUrl string `json:"html_url"`
```

Comment:
```suggestion
	HTML_URL string `json:"html_url"`
```
Maybe?

Reply:
This was a already created structure i have just changed the position i don't think we should change this because then i have to change the code associated with this structure as well.